src/modm/platform/usb/rp/module.lb

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2022, Andrey Kunitsyn
+#
+# This file is part of the modm project.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+# -----------------------------------------------------------------------------
+
+from collections import defaultdict
+
+
+# -----------------------------------------------------------------------------
+def init(module):
+    module.name = ":platform:usb"
+    module.description = "Universal Serial Bus (USB)"
+
+def prepare(module, options):
+    device = options[":target"]
+    if not device.has_driver("usb:rp20*"):
+        return False
+
+
+    module.depends(
+        ":architecture:interrupt",
+        ":cmsis:device",
+        ":platform:gpio",
+        ":platform:core")
```

Comment:
Don't depend on the core module in drivers, only the `project.xml` does that (this allows the startup code to be replaced externally if required)